programs/shfmt.nix

```
@@ -6,12 +6,20 @@ in
   options.programs.shfmt = {
     enable = lib.mkEnableOption "shfmt";
     package = lib.mkPackageOption pkgs "shfmt" { };
+    indent_size = lib.mkOption {
+      type = lib.types.nullOr lib.types.int;
+      default = 4;
```

```suggestion
      default = 2;
```
After some more thought, let's keep that at 2 to avoid breaking back-compat.